plugin/src/software/aws/toolkits/eclipse/amazonq/chat/ChatCommunicationManager.java

```diff
@@ -148,8 +149,7 @@ private ChatRequestParams addEditorState(final ChatRequestParams chatRequestPara
         // only include files that are accessible via lsp which have absolute paths
         getOpenFileUri().ifPresent(filePathUri -> {
             chatRequestParams.setTextDocument(new TextDocumentIdentifier(filePathUri));
-            getSelectionRangeCursorState()
-                    .ifPresent(cursorState -> chatRequestParams.setCursorState(Arrays.asList(cursorState)));
+            getSelectionRangeCursorState().ifPresent(cursorState -> chatRequestParams.setCursorState(Arrays.asList(cursorState)));
```

Comment:
please revert any formatting changes in this PR. That takes away time from reviewing the actual change

Reply:
Okay, will do. Sorry!